Sources/TestingMacros/ConditionMacro.swift

```diff
@@ -435,7 +435,7 @@ extension ExitTestConditionMacro {
 
     // TODO: use UUID() here if we can link to Foundation
     let exitTestID = (UInt64.random(in: 0 ... .max), UInt64.random(in: 0 ... .max))
-    let exitTestIDExpr: ExprSyntax = "Testing.__ExitTest.ID(__uuid: (\(literal: exitTestID.0), \(literal: exitTestID.1)))"
+    let exitTestIDExpr: ExprSyntax = "(\(literal: exitTestID.0), \(literal: exitTestID.1))"
```

Comment:
Is the switch to a tuple because the ID type is now `@_spi`?

Reply:
Yes, it reduces the number of symbols we need to make `public`. The current protocol-based discovery mechanism will go away once we get `@section`, and the PR for that should make it more clear how we consume this value.